deb/debian/install

```diff
@@ -1 +1 @@
-debian/collectd.conf opt/stackdriver/collectd/etc/
```

Comment:
This would require us to change all of our documentation, so it's probably best to do it after we change the versioning scheme, as part of a major version bump... @qingling128 @bmoyles0117 WDYT?

Reply:
What if we supported both the old and the new locations simultaneously? Then legacy users wouldn't see issues and those that wanted to use best practices would use the new mechanism?

We'd still have to update documentation, but wouldn't break existing users...would we?

Reply:
Collectd does not support multiple config files, but users aren't supposed to touch or care for this file anyway. We can definitely support two locations for user-specified config files, though, as long as they use only one of them at a time.
This bit is just to ease the upgrade for power users who touched the template because we've told them to.

Reply:
I'm curious to see how they handled upgrades in the existing scenario. Any user updates to this template are blown away each time. I don't think this change would break anyone anymore that it already does.

Reply:
Exactly. Changes to the template only last until the next upgrade.

Reply:
SGTM.

Reply:
Agreed, especially with the consideration how upgrade behaves today (blow away older configurations regardless).